.github/workflows/ci.yml

```diff
@@ -33,6 +33,13 @@ jobs:
 
       - name: Check for forbidden words
         run: "! grep --include='*.rs' -RE 'to_be_bytes|from_be_bytes|dbg!' ."
+        if: runner.os != 'Windows'
+
+      - name: Check for forbidden words
+        run: |
+            findstr /s "to_be_bytes from_be_bytes dbg!" *.rs
+            if ("%ERRORLEVEL%" -ne "1") { exit "%ERRORLEVEL%" }
+        if: runner.os == 'Windows'
```

Comment:
Lets remove this. The CI always runs the unix check, and since it's a regex there's no need to run it again on Windows, and I'd rather not have to add things to both lists

Reply:
oh ok i thought i'd better delete it but i thought i should keep it consistent

I was the first to read the powershell documentation to learn this and it was helpful.
The github window did a great job of shoveling powershell, not cmd.

Reply:
<https://github.com/cberner/redb/pull/379/commits/80a43d8c9e2af3a014175fa25cfd106c8ea71967>
here! I edited